OreNoTask/app/controllers/admin/users_controller.rb

```diff
@@ -47,6 +47,13 @@ def create
     end
 
     def destroy # rubocop:disable Metrics/AbcSize
+      @user = User.active.find_by(id: params[:id])
+
+      if @user.privilege == 1 && User.admin_user_count == 1
+        redirect_to admin_users_path, notice: I18n.t('dictionary.messages.last_admin')
+        return
+      end
```

Comment:
【修正任意】
`return`と`redirect_to`については以下のように1行で書き換えても大丈夫です。
どちらでも大丈夫なのでお好みでどうぞ！

```suggestion
      if @user.privilege == 1 && User.admin_user_count == 1
        return redirect_to admin_users_path, notice: I18n.t('dictionary.messages.last_admin')
      end
```

Reply:
修正しました（知らなかったのでやってみたかった
https://github.com/Fablic/training/pull/1187/commits/8668aa20a30e7aae06ea4edaafd274f5091ae0bf